test_pytest_html.py

```diff
@@ -261,6 +255,31 @@ def pytest_runtest_makereport(item, call):
             href)
         assert link in html
 
+    def test_extra_text_separeted(self, testdir):
+        content = str(random.random())
+        testdir.makeconftest("""
+            import pytest
+            @pytest.mark.hookwrapper
+            def pytest_runtest_makereport(item, call):
+                outcome = yield
+                report = outcome.get_result()
+                if report.when == 'call':
+                    from pytest_html import extras
+                    report.extra = [extras.text('{0}')]
+        """.format(content))
+        testdir.makepyfile('def test_pass(): pass')
+        result, html = run(testdir)
+        assert result.ret == 0
+        hash_key = ('test_extra_text_separeted.py::'
```

Comment:
How we can access the request from the test? And I think it is a good way to assert that the tests are being generated the way we want: the test id, plus the id of the test and the id of the fixture. I think if we just get it from the request, we could not assert that it is being generated by the key we want.